### ISSUE TITLE
The import of audiogen at Colab

### ISSUE THREAD

**Issue by underdogliu** [facebookresearch/audiocraft#265]
Hi and first of all thanks for such amazing work! 

So I am now starting from the very beginning by trying the model via Colab. I found that when trying to import the audiogen model and run the related example script [link](https://github.com/facebookresearch/audiocraft/blob/main/docs/AUDIOGEN.md), I have to install via the "bloody edge" method:
```
pip install -U git+https://git@github.com/facebookresearch/audiocraft#egg=audiocraft
```
instead of simply "pip install -U audiocraft". Is this something expected? Or I got it wrong somewhere in the middle?

Below is my example script which shall be run on Colab with T4.
```
%%capture
!pip install torch
!pip install -U git+https://git@github.com/facebookresearch/audiocraft#egg=audiocraft

import audiocraft
from audiocraft.models.audiogen import AudioGen
from audiocraft.data.audio import audio_write
```

**Comment by adefossez** [comment]
Yes at the time we had not release the stable version of audiocraft updated with all the new models. Now it should work though as we officially released the v1.0.0 on PyPI.
